src/useSpeech/index.ts

```diff
@@ -0,0 +1,117 @@
+import { useCallback, useEffect, useRef, useState } from 'react';
+import { useIsMounted } from '../useIsMounted';
+
+type SpeechOptions = {
+	voice?: SpeechSynthesisVoice;
+	rate: number;
+	pitch: number;
+	volume: number;
+	lang?: string;
+};
+
+export type ISpeechOptions = Partial<SpeechOptions>;
```

Comment:
I dont see any sense in making a type and then making a new one with interface prefix.

Moreover we dont use these prefixes anymore.

Reply:
Addressed [here](https://github.com/react-hookz/web/pull/1378/commits/f2934d4809c520d72d0bc72ffe8938da8ac1ba08)

Remove all types starting with `I` and added all the types only for necessary state.